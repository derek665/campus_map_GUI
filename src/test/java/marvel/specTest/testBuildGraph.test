# test the correct graph is being built

LoadGraph testBuild soccer.tsv

ListNodes testBuild

ListChildren testBuild Barcelona
ListChildren testBuild Arsenal