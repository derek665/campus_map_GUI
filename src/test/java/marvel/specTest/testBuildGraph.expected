# test the correct graph is being built

loaded graph testBuild

testBuild contains: Arsenal Barcelona Eibar Juventus Liverpool Marseille PSG Southampton Valencia

the children of Barcelona in testBuild are: Eibar(La-Liga) Juventus(Champions-League) PSG(Champions-League) Valencia(La-Liga)
the children of Arsenal in testBuild are: Liverpool(EPL) Southampton(EPL) Valencia(European-League)